app/viewmodels/models/Flag.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewmodels.models
+
+import play.api.libs.json.{Json, OFormat}
+
+// Used in UserAnswers as a Unit type - where the value is not important
+case object Flag {
+  implicit val format: OFormat[Flag] = Json.format[Flag]
```

Comment:
required as a unit value would also need a custom read and writes to be created - this just seemed easier and the naming fits our use cases